inc/agl.inc

```diff
@@ -4,53 +4,31 @@ LICENSE = "MIT"
 
 inherit build_yocto
 inherit xt_quirks
-require xt_rcar.inc
 
 S = "${WORKDIR}/repo"
-# path for machine specific layers and recipes
-FILESEXTRAPATHS_prepend := "${THISDIR}/../machine:"
+
+################################################################################
+# set to AUTOREV so we can override this while reconstructing this build with
+# specific commit ID
+################################################################################
+SRCREV = "${AUTOREV}"
 
 # path for machine non-specific recipes
 FILESEXTRAPATHS_prepend := "${THISDIR}/../recipes-domx:"
 
 XT_QUIRK_UNPACK_SRC_URI += " \
     file://meta-xt-images-domx;subdir=repo \
-    file://meta-xt-images-vgpu;subdir=repo \
 "
 
 # these layers will be added to bblayers.conf on do_configure
 XT_QUIRK_BB_ADD_LAYER += " \
     meta-xt-images-domx \
-    meta-xt-images-vgpu \
 "
 
-################################################################################
-# set to AUTOREV so we can override this while reconstructing this build with
-# specific commit ID
-################################################################################
-SRCREV = "${AUTOREV}"
-
 ################################################################################
 # AGL has its own setup script which calls poky's one internally
 ################################################################################
-XT_AGL_FEATURES = "agl-appfw-smack"
+XT_AGL_FEATURES ?= "agl-appfw-smack"
```

Comment:
This change also is not covered in the commit message.

Reply:
will remove